src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -265,28 +267,30 @@ const ReleaseContent = () => {
                   {release.coverageTitle} <strong>{release.yearTitle}</strong>
                 </p>
                 <Details summary={`See other releases (${releaseCount})`}>
-                  <ul className="govuk-list">
-                    {[
-                      ...release.publication.releases.map(
-                        ({ id, title, slug }) => (
-                          <li key={id} data-testid="other-release-item">
-                            <Link
-                              to={`${config?.PublicAppUrl}/find-statistics/${release.publication.slug}/${slug}`}
-                            >
-                              {title}
-                            </Link>
-                          </li>
+                  <ScrollableContainer maxHeight={300}>
+                    <ul className="govuk-list" style={{ overflowY: 'scroll' }}>
```

Comment:
The overflow should be applied in the `ScrollableContainer` component (see other comment)